executor/executor_test.go

```diff
@@ -5663,9 +5663,10 @@ func TestAdmin(t *testing.T) {
 	tk.MustExec("use test")
 	tk.MustExec("drop table if exists admin_test2")
 	tk.MustExec("create table admin_test2 (c1 int, c2 int, c3 int default 1, index (c1))")
+	// we cn only see last 10 queries
```

Comment:
```suggestion
	// we can only see last 10 queries
```

Due to BatchCreateTableWithInfo, we have less history DDL jobs